docs/guide/en/accordion.md

```diff
@@ -0,0 +1,210 @@
+# Accordion
+
+The Accordion widget renders a Bootstrap 5 accordion component within the **Yii framework**.
+
+You can use it to create collapsible content sections where clicking a header toggles the visibility of its associated
+body. 
+
+This component is highly customizable, supporting options like always-open behavior, flush styling, custom HTML tags,
+and CSS classes.
+
+## Key Features
+- Supports multiple collapsible items with headers and bodies.
+- Configurable to allow multiple items to remain open simultaneously (`alwaysOpen`).
+- Customizable header and toggler tags, attributes, and styles.
+- Optional flush styling to remove borders and rounded corners.
+- Flexible HTML attribute and CSS class management for container, header, body, and toggler elements.
+
+## Quick Start
+To get started, instantiate the `Accordion` widget and add items to it. Here is a basic example:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Bootstrap5\Accordion;
+use Yiisoft\Bootstrap5\AccordionItem;
+?>
+
+<?= Accordion::widget()
+    ->items(
+        AccordionItem::to('Item #1', 'This is the first item\'s content.'),
+        AccordionItem::to('Item #2', 'This is the second item\'s content.'),
+        AccordionItem::to('Item #3', 'This is the third item\'s content.'),
+    )
+?>
+```
+
+This generates a simple accordion with three collapsible items, each with a clickable header and a body that
+expands/collapses.
+
+## Basic Usage
+
+### Instantiation
+Create a new `Accordion` instance using the `widget()` method:
+
+```php
+$accordion = Accordion::widget();
+```
+
+### Adding Items
+Use the `items()` method to define accordion items. Each item is an instance of `AccordionItem`:
+
+```php
+$accordion = Accordion::widget()
+    ->items(
+        AccordionItem::to('Section 1', 'Content for section 1.'),
+        AccordionItem::to('Section 2', 'Content for section 2.', active: true),
+        AccordionItem::to('Section 3', 'Content for section 3.'),
+    );
+```
+
+### Rendering
+Render the accordion as HTML using `render()` or cast it to a string:
+
+```php
+// Using render()
+$html = $accordion->render();
+
+// Using __toString()
+$html = (string) $accordion;
+```
```

Comment:
Overall it should be close to what's actually used in HTML and I don't see a need to separate these into separate sections. Could be a single example since typically all these come together.

Reply:
As i explained earlier, we can make a single example like this in all the guides, but each widget has different methods. The idea is to show it in a simple and quite explicit way, this will allow the documentation to be clear enough for everyone.

Reply:
I don't agree. I think we need compact examples that are close to real world usage.

Reply:
The closest thing to a real case would be to include the main example from the bootstrap5 documentation.

Reply:
I can delete the sections, do you just want to leave a single example and that's it?